src/Moq/Setup.cs

```diff
@@ -21,6 +21,8 @@ protected Setup(InvocationShape expectation)
 
 		public virtual Condition Condition => null;
 
+		public InvocationShape Expectation => this.expectation;
+
 		public LambdaExpression Expression => this.expectation.Expression;
```

Comment:
Should check whether `Expression`, `Method` still need to be exposed at all now that they'd be accessible through `Expectation`. It might be convenient to keep them around.